coderd/coderd.go

```diff
@@ -819,6 +819,10 @@ func New(options *Options) *API {
 		httpmw.CSRF(options.SecureAuthCookie),
 	)
 
+	if buildinfo.IsDev() {
+		r.Use(httpmw.RecordAuthzChecks)
+	}
```

Comment:
It seems like this would also be useful to enable in tests?

Reply:
builds run during tests should still be dev builds, right?

Reply:
oops yes you're right